### PR TITLE
Globo-Slow

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Increased movement delay across the board by 15%.
Marines no longer get light armor by default, but it can be ordered by Req from ASRS. It costs slightly more now.
Heavy armor now has 4 storage slots.

## Why It's Good For The Game

Everything in CM is extremely fast-paced, slowing it down a notch will make rounds last longer, fights be less of two people dancing across the screen, and people will be more inclined to stay in one location and talk.

Light armor is just the best armor marines can possibly take, as speed is king. By taking it out, marines can now make a choice between a bunch of equally-good armor sets. If they really want it that badly, they can order it from Req.

Heavy armor kinda sucks, so giving it more storage makes it better.

## Changelog

:cl:
balance: Increased movement delay across the board by 15%.
balance: Marines no longer get light armor by default, but it can be ordered by Req from ASRS. It costs slightly more now.
balance: Heavy armor now has 4 storage slots.
/:cl:

### CODE REVIEW

**Comment by stanalbatross** [comment]
No

**Comment by JohnDoe50974** [comment]
Is this the thing that causes a dev team schizm

**Comment by Shad0vvs** [comment]
🍿

**Comment by kiVts** [comment]
this is not how you slow down the game

**Comment by usnpeepoo** [comment]
somebody get kineems thoughts on this i remember his rant about xeno/marine speeds a while back.

**Comment by Nanu308** [comment]
50% why by allah

You get to drop it to 15% and no more, and we can test that.

Re-open PR when it's done.

**Comment by Don2137** [comment]
Geeves i think putting us at aurora walk speed is slightly too far

**Comment by hotimep** [comment]
mfw legs break and i'm wearing heavy armor and i walk at 2 tiles a minute

**Comment by stanalbatross** [comment]
changing movement speed at all requires basically the entire game to be rebalanced. Bullets suddenly become way faster in relation to you, it takes xenos way longer to get in and out of fights (meaning that they have way more time to be shot at and thus die). It makes CAS and OBs faster compared to xenos, giving them less time to get out of range and making them more likely to die to them.

`slowing it down a notch will make rounds last longer`

really not the right way to do this. PRs like static comms and IOs which actually give CONTENT to improve round pacing are the way to go, not this.

`people will be more inclined to stay in one location and talk.`

they'll just stay in one location and shoot from a bipodded HPR because that gun will become even more powerful if xenos have 30% more time in its LOS when attacking a cade

arguably, when they do move, people will have to spend even MORE time moving, cutting down on downtime for roleplay

or the game will just devolve into cave and cade hugging because neither side can push (n.b. this makes cades incredibly strong. 30% more time in the LOS of the defenders.)

concerns are not just about xenos though - marines will die MUCH more in boiler gas, oppressor hooks, and other things that they can't get out of the way of in time.

`By taking it out, marines can now make a choice between a bunch of equally-good armor sets. `

two of them? heavy armour will not become equally good at all - storage is already easy enough and having 1 more slot will barely benefit them.

**conclusion**

reduction in movement speed would massively upset game balance, turning the game into a cadehug slog and snoozefest. Fast paced combat and movement is FUN. trying to "incentivise RP" is not worth losing a very very major part of what makes CM enjoyable. This PR fails at its aims and fails to improve the gameloop/gameplay of CM at all. If you want to make longer rounds, fill them with meaningful content not dead time spent moving or cadehugging or waiting for the dropship to charge

**Comment by Don2137** [comment]
> changing movement speed at all requires basically the entire game to be rebalanced. Bullets suddenly become way faster in relation to you, it takes xenos way longer to get in and out of fights (meaning that they have way more time to be shot at and thus die). It makes CAS and OBs faster compared to xenos, giving them less time to get out of range and making them more likely to die to them.
> 
> `slowing it down a notch will make rounds last longer`
> 
> really not the right way to do this. PRs like static comms and IOs which actually give CONTENT to improve round pacing are the way to go, not this.
> 
> `people will be more inclined to stay in one location and talk.`
> 
> they'll just stay in one location and shoot from a bipodded HPR because that gun will become even more powerful if xenos have 30% more time in its LOS when attacking a cade
> 
> arguably, when they do move, people will have to spend even MORE time moving, cutting down on downtime for roleplay
> 
> or the game will just devolve into cave and cade hugging because neither side can push (n.b. this makes cades incredibly strong. 30% more time in the LOS of the defenders.)
> 
> concerns are not just about xenos though - marines will die MUCH more in boiler gas, oppressor hooks, and other things that they can't get out of the way of in time.
> 
> `By taking it out, marines can now make a choice between a bunch of equally-good armor sets. `
> 
> two of them? heavy armour will not become equally good at all - storage is already easy enough and having 1 more slot will barely benefit them.
> 
> **conclusion**
> 
> a reduction in movement speed would massively upset game balance, turning the game into a cadehug slog and snoozefest. Fast paced combat and movement is **FUN**. trying to "incentivise RP" is not worth losing a very very major part of what makes CM enjoyable. This PR fails at its aims and fails to improve the gameloop/gameplay of CM at all. If you want to make longer rounds, fill them with meaningful content not dead time spent moving or cadehugging or waiting for the dropship to charge
> 
> As such I am completely opposed

I actually don't agree that lower speed makes the game less fun, quite the opposite. Remember, you can only see 7 lengths of your character in any direction. This means that the faster people move, the more likely they are to walk into an unfavorable situation and die.

As for cade/cavehugging, this is specifically (and somewhat exclusively) caused by rifle damage creep. M41 just keeps getting better and every spam fire weapon is brought up to its level (while stuff like shotguns or flamer remain situational). The game is basically TGMcifying itself.

I do agree this change would upset everything though (which isn't inherently bad). Make a design doc.

**Comment by kiVts** [comment]
movement delay is going to be horribly noticeable and just too much uncomfortable, only if we had movement  animation slowdown....

**Comment by Erawh** [comment]
laggy rounds would be hell, 15 minutes to walk from fob to the front, marines would get significantly stronger and would be discouraged from pushing out. 
Entire game would have to be rebalanced around the global slow, for a change that would only feel bad to play in.

Armors do need a rework or some kind, to make the non light ones more playable, but this isnt the way to do that.

**Comment by neeshacark** [comment]
good god no this is going to piss so many people off and nobody has ever complained about it.

**Comment by hotimep** [comment]
actually i changed my mind if everything gets slowed it will make it 10x easier for me to aim at sexnos add it in

**Comment by optimisticdude** [comment]
i kneel
zazed

**Comment by memesky117** [comment]
get this man a speedmerge

**Comment by Kuusday** [comment]
This ain't it. You'd have to completely rebalance stuff like bullet speed and xeno caste time for abilities. Warriors for example will have an easier time lunging and oppress hook will snatch people up simply because of the increased movement delay.

As for armor, Heavy getting two additional storage slots won't really do much. Heavy currently exists as drip since if you get a leg frac in light armor you're as slow as heavy. I don't think removing light armor from vendors is the right call.

**Comment by Diegoflores31** [comment]
"lets remove 1 of the only 3 armors available yay"
marines will literally just start going no armor or jacket

**Comment by Don2137** [comment]
> "lets remove 1 of the only 3 armors available yay" marines will literally just start going no armor or jacket

One of the 3 armors, except it's the only one worth using in any situation. Removing light unironically increases the variety by virtue of getting rid of the only right choice.

**Comment by Megarop** [comment]
resist globoslowmo. i will not cade the hugs

**Comment by AGasCanister** [comment]
ok give heavy armor feet protection if you want marines to use other types of armor instead of going with nothing :D

**Comment by Aether217** [comment]
This is it boys, turn based movement

**Comment by Erawh** [comment]
> ok give heavy armor feet protection if you want marines to use other types of armor instead of going with nothing :D

straight up just make a rework of the armors entirely, light is all everyone who knows anything uses and the others just cant compete, bandaid solutions like changing a stat or two on heavy wont make people switch if fundamentally light is just better

**Comment by gnorse** [comment]
Damage isn't what kills you, stuns do, and light armor sure is a lot better at helping you avoid stuns than heavy.

**Comment by Marrone321** [comment]
Compared to other servers, CM is already relatively slow. I remember playing a bunch of CM, jumping on some TG servers, and thinking holy shit I am fucking fast. I would really like to not always pick light armor, so very much in favor of adding benefits such as limb/leg protection to heavy or making it have a chance to resist stuns, just anything to give more nuance to the three armors would be a welcome change and allow us to tailor them to our loadouts and roles.

With that said, I dont really agree with removing the light armor, not because its the best, but because the options for armor are already quite limited. Perhaps nerfing light armor, and buffing the other two in order to even them out may be the right move, but as it is light armor has already pretty minimal protection against the xenos, and very small storage space - its big thing going for it is the ability to manuever, which for any squad even Bravo is important in order to relocate, retreat, avoid xeno abilities, etc. 

As for the slowdown, if its something you're really dedicated to, I would advise something far more marginal than 15%... as a fraction 15% = 3/20, which is not _quite_ small enough for something that would affect the flow of game. Perhaps something like 3% or so, even if its a change subjectively I wouldnt want, I think there is a compromise if you're really honed in on that.

However I agree with Stan, I feel like there are many other ways to get players to slow down then literally lowering the movement speed, things that require players to communicate and think about their context before making impulsive decisions.

Definitely like the out-of-the-box thinking here, but its a bit on the nose - perhaps more time needs to pass before it should be considered again in the future.

Regardless of all the criticism Geevies, I hope I speak for everyone when I say we really appreciate your contributions and I look forward to what you got in store for us, I just ask you do not get discouraged and take in feedback and deliver some awesome content!

**Comment by GoldenDarkness55** [comment]
Based as fuck, cringe light armor and MS creep should have been done away with ages ago.

**Comment by RadiationFan** [comment]
Honestly, this seems more like an issue with the paincrit system. Rebalancing the debuffs marines get for taking damage and broken bones would be a better way to slow the pace down of rounds.

 Having people get broken legs and groins, and just stabbing themselves with oxy and continuing to fight is pretty ridiculous. If you have a compound fracture in your leg, you are not likely going to walk more than twenty feet before passing out from pain, yet marines regularly break half of their bones and just keep trucking.

 If heavy armor actually ended up being faster most of the time, for example, heavy armor gets lurked, no bones broken, still moving at the same speed, where as a light armor boy gets bone broken and is now moving at the speed of a snail, heavy armor would become much more attractive. 

Aswell, increasing the punishment for spreading out and ungaing would likely lead to better squad cohesion, after a while, due to the fact that if the marines do not work together and run off wearing a speedo and two mou53s, the first runner that breaks PFC Ungas foot will drag him to brazil, where as PFC RP, wearing heavy armor and hanging out with his battle buddies, will not suffer the fate of a broken leg, and will continue to get to shoot at xenos. 

To conclude my XenoU talk, pain, and its consequences, are the reason light armor is so attractive. If you made taking damage an actual issue, having more protection would actually matter. As it stands, as long as the only thing that matters is movement speed and not having your hands or arms broken and unsplinted, marines will continue to use light armor. If taking damage had significant consequences, heavier, more effective armors would matter again, but as long as pain is neutered, light armor will remain the superior option for survivability, combat effectiveness, and marine quality of life during operations.

**Comment by kiVts** [comment]
> Honestly, this seems more like an issue with the paincrit system. Rebalancing the debuffs marines get for taking damage and broken bones would be a better way to slow the pace down of rounds.
> 
> Having people get broken legs and groins, and just stabbing themselves with oxy and continuing to fight is pretty ridiculous. If you have a compound fracture in your leg, you are not likely going to walk more than twenty feet before passing out from pain, yet marines regularly break half of their bones and just keep trucking.
> 
> If heavy armor actually ended up being faster most of the time, for example, heavy armor gets lurked, no bones broken, still moving at the same speed, where as a light armor boy gets bone broken and is now moving at the speed of a snail, heavy armor would become much more attractive.
> 
> Aswell, increasing the punishment for spreading out and ungaing would likely lead to better squad cohesion, after a while, due to the fact that if the marines do not work together and run off wearing a speedo and two mou53s, the first runner that breaks PFC Ungas foot will drag him to brazil, where as PFC RP, wearing heavy armor and hanging out with his battle buddies, will not suffer the fate of a broken leg, and will continue to get to shoot at xenos.
> 
> To conclude my XenoU talk, pain, and its consequences, are the reason light armor is so attractive. If you made taking damage an actual issue, having more protection would actually matter. As it stands, as long as the only thing that matters is movement speed and not having your hands or arms broken and unsplinted, marines will continue to use light armor. If taking damage had significant consequences, heavier, more effective armors would matter again, but as long as pain is neutered, light armor will remain the superior option for survivability, combat effectiveness, and marine quality of life during operations.

nah bud

**Comment by AnCopper** [comment]
> MOBA

is just stupid how unless you have 40 macros set up and 3 wifi routers, you can't win as a MOBA xeno, combo gameplay completely shits on xenos and doesn't work in SS13 at all. Especially when all marines have to do is 'unga point and click', 0 skill requirement. Most xeno strains now need a PhD in that exact strain and the sheer grace of god to do well with CM's lag. These marine biased dev's just keep fucking around with xeno's meta thinking they can turn the gameplay into Smash Bro's Ultimate without playing a single xeno round themselves. A year ago xenos actully had some weight to them but now I gotta do A+B+X+Y+Z+A+Y Combo move to fucking sneeze on a marine let alone do: any damage. This shit aint working on a 20 year old 2d space man game that turns into a powerpoint slideshow after 10 minutes of gameplay.

**Comment by Don2137** [comment]
This PR makes ghosts waaaaaaaaay too slow

**Comment by Mister-moon1** [comment]
This pr makes me want to shove toothpicks into my toenails and kick a wall

**Comment by ManoPablo** [comment]
nope this ain't it chief, just way too god damn slow.
6ra86j

**Comment by Kuusday** [comment]
Right. As expected, this was terrible.

Genuinely unfun and gameplay felt super tedious. The only compromise I can see working is removing light armor and not touching slowdown. Otherwise people will be snails.

**Comment by stanalbatross** [comment]
played a round with this and confirmed all my prior thoughts

- HPRs were very, very strong. Xenos had a much harder time getting out of my LOF due to the speed reduction and as such they got hit a lot especially by holotargeting rounds
- OBs and explosives in general were much stronger, less time to dodge 
- marines were affected more too because of shitty OT nades being undodgeable
- runner and lurker are useless now that marines can easily mow them down 
- queen hella slow
- it affects GHOSTS
- much harder to get out of the way of boiler telegraphs / crusher charge telegraphs

predrop it wasn't too bad but once we landed it was noticeable

also regards to removal of light armour, this is just pointless. Marines will powergame whatever faster movement speed they can get and it doesn't actually achieve the goal (? if there even was a goal in the first place) of making the other armour types better / more viable / a better pick as people will just go for medium anyway as the slowdown of heavy will guarantee your death still. 1 extra storage slot is NOT enough.

**conclusion**

despite a pretty good xeno side and competent players in the first test, through the slog of slow HPR and folding cade pushes we got to the hive as well as via fire support.

Xenos' attempts to flank were restricted by their slow movement speed and the fact we could hear the queen 3 minutes before she came into our LOS, then spam her down with HPRs.

negative change in every aspect.

**Comment by Waseemq1235** [comment]
This PR just upset every single aspect of the game balance so damn hard that even if every single fucking deity from every fucking religion in existance came down they wouldn't be able to fix it and decide to end the world there/
To name a few:

- fast based xeno castes are, needless to say, literally fucked. runner moves at the speed of a warden prae, with the health of a toothpick. you took an already shitty caste and made it a meme caste now.
- no marine light armor? defenders and warriors now outrun you off weeds, no pheros. i dont need to add more to this honestly.
- rip ghosts, might aswell not observe
- queen is now a fucking screaming wall that happens to be able to move every so often
- CM is already really slow compared to other servers, you might aswell make CM a turn based game on a checkerboard, because that's what it is with this PR
- i cant even imagine MPs with their armor, its already literal hell without this PR, now i might aswell beat my spine with a sledgehammer to get cerebral paralysis, same thing
- rip preds, their only true tank is their speed, you might aswell be able to direct hit them with a SADAR
- oh boy i cant imagine how all of the above is gonna get even worse at weekend-pops
- take this shit with a pinch of salt, its meant to be a copypasta

**Comment by Don2137** [comment]
Funny how the "cm is already slow compared to other servers" crowd is always comparing to lrp servers and not roleplay ones.............

**Comment by RadiationFan** [comment]
Good change, made combat as xeno and survivor feel better, runners nolonger move at what approaches light speed, marines did not unga of on their own so much. 1vX as xeno felt better, as marines could nolonger gap you and gun you down. 

Bad parts-Gunlines are now quite strong, but has been the case for quite some time. 

Overall, I like it, I feel like light armor should be returned, but make it more of a risk-reward playstyle. Remove internalprotection and make it even faster than it currently is, but when you take damage to your body, you are going to get messed up. 

Perhaps even a -internalprotection modifier, to increase the risk/reward gameplay style. It could be maybe that the armor was poorly designed and when struck, spalls from the inside in to your organs. We even have a canon arguement for this with the M2C, where it was made poorly and over heats. The M3L could just be made in such a way where canonnically, it is a deathtrap as far as armor is concerned, useful for people with skill and agility, dangerous for people that think they have skill and agility. 



> played a round with this and confirmed all my prior thoughts
> 
> * HPRs were very, very strong. Xenos had a much harder time getting out of my LOF due to the speed reduction and as such they got hit a lot especially by holotargeting rounds
> * OBs and explosives in general were much stronger, less time to dodge
> * marines were affected more too because of shitty OT nades being undodgeable
> * runner and lurker are useless now that marines can easily mow them down
> * queen hella slow
> * it affects GHOSTS
> * much harder to get out of the way of boiler telegraphs / crusher charge telegraphs
> 
> predrop it wasn't too bad but once we landed it was noticeable
> 
> also regards to removal of light armour, this is just pointless. Marines will powergame whatever faster movement speed they can get and it doesn't actually achieve the goal (? if there even was a goal in the first place) of making the other armour types better / more viable / a better pick as people will just go for medium anyway as the slowdown of heavy will guarantee your death still. 1 extra storage slot is NOT enough.
> 
> **conclusion**
> 
> despite a pretty good xeno side and competent players in the first test, through the slog of slow HPR and folding cade pushes we got to the hive as well as via fire support.
> 
> Xenos' attempts to flank were restricted by their slow movement speed and the fact we could hear the queen 3 minutes before she came into our LOS, then spam her down with HPRs.
> 
> negative change in every aspect.

1.HPR was already strong, this just made it better, whatever, increase damage falloff on HPR would help, atleast with the rundown aspect of getting mulched offscreen by gunlines.
2. Making firesupport better is, in general, a good thing. 
3. OT nades are a question unto themselves, the fact that pocket OBs are manufactured should be looked at.
4. Lurker suffers from map prejudice, LV, lurker rocks, good places to break firelines, good places to kite, good tile colors to stalk, Big Red sucks ass, Lurker camo blows ass, unless you are on weeds, but then you could just play warrior and be 10x more useful anyway.
5. Then make queen less slow, or remove the loud, fuckoff stomps, or reduce the range down to one screen instead of like three. Or give shaman an ability to mimic queen stomps when the utility shaman PR goes back up. Alot can be done, being slow is not in and of itself the only thing the queen has going wrong.
6. Yeah over sight is bad, but Dchat is not really part of the gameplay loop anyway.
7. Xeno fire support and crusher have every right to exist, just like marine fire support.
I agree with you on all these things, but globoslomo only exacerbates issues that already exist, the only difference is instead of dying in the the door trying to get to cover from a caded gunline, you die two or three tiles away. And getting filtered by firesupport will hopefully make rounds more engaging, and create more attentive players. But I am a noname pleb, I will go back to by corner and shut up now.

**Comment by gnorse** [comment]
> Funny how the "cm is already slow compared to other servers" crowd is always comparing to lrp servers and not roleplay ones.............

yeah because lrp servers tend to have more combat. Why would you care about the speed of a server who's entire playerbase is sitting at a bar talking

**Comment by Listaless** [comment]
Personally a Fan of the 15% slowdown, honestly don't care either way about light armour removal. I know people are complaining but the slower feeling makes the game feel more tactical.

**Comment by c4xmaniac** [comment]
People are making it sound a lot worse than it actually is. It really wasn't that bad.

You would need to change other stuff to go along with it, greenfire, HPR, SG, resin crack, pounce CD, etc but I do not think a slower paced game is necessarily worse than what we have at the moment. Rightfully, people were extra worried about how much this would affect xenos, but considering they just won with a 40+ hive before hijacking on 140 pop, I think xenos did pretty well with Yellows queen. 

I don't know how the dev team wants to go forward, especially since the RP debate is the most autistic thing imaginable, but slowing gameplay leads to decisions mattering more and giving more time/opportunity to RP. Last I heard was that CM is a TDM game with adjacent RP elements, and if that is how the devs want to view CM (TDM aspect first) then this change probably shouldn't make it through. Though, a movement change more in line with TGMC would be a step forward in that regard.

**Comment by anclandor** [comment]
I feel like this slowdown will hurt roleplay a lot.  Right now you are unable to talk and walk, so getting left behind by your battlebuddies as you are talking a big issue (though typing indicators do help).  This slowdown makes this issue even worse.  If walking somewhere takes more time due to slowdown, than the opportunity cost of the movement you are giving up to type increased.  
tldr - time spent walking increased = time spent talking decreased

**Comment by Waseemq1235** [comment]
> People are making it sound a lot worse than it actually is. It really wasn't that bad.
> 
> You would need to change other stuff to go along with it, greenfire, HPR, SG, resin crack, pounce CD, etc but I do not think a slower paced game is necessarily worse than what we have at the moment. Rightfully, people were extra worried about how much this would affect xenos, but considering they just won with a 40+ hive before hijacking on 140 pop, I think xenos did pretty well with Yellows queen.
> 
> I don't know how the dev team wants to go forward, especially since the RP debate is the most autistic thing imaginable, but slowing gameplay leads to decisions mattering more and giving more time/opportunity to RP. Last I heard was that CM is a TDM game with adjacent RP elements, and if that is how the devs want to view CM (TDM aspect first) then this change probably shouldn't make it through. Though, a movement change more in line with TGMC would be a step forward in that regard.

you said it  yourself: yellows queen, above that, yellows was eating alacrit fruit like a fucking meth addict to counter the slow so yeah

**Comment by Tisx1** [comment]
As an observer main and vet player who has seen slowdowns before (granted not on this scale) I guess I show put in my two cents. 

Firstly. This is a hard xeno nerf and a massive marine buff. All of the Marines useful weaponry is based off of ranged combat, with the majority of marine weapons being able to reach across the entire screen. Meanwhile the majority of xeno castes are based off of melee and/or stun combat, xenos have 3 castes that are able to shoot at screen length, boiler, prae and spitter, (Boiler can only fire once and have a long wind up, Prae and spitter start struggling to hit anything past 5 tiles). Now xenos do have screen length abilities but they are movement abilities, which move the mob itself. 

With this in mind we can already see that marines will be less affected by movement slowdown, yes, they may be slightly slower. But the bullets aren't, all of the guns are still firing the exact same way they were before this, and that is this PRs greatest flaw. **Any form of xeno slowdown is free bonus damage for marines. Allow me to show you why.**

Lets say we have one marine named PVT Unga Bald with a M41 and he is chasing a T3 Crusher

Before this PR let's say it takes the Crusher 6 seconds to break contact from PVT Unga Bald

After the PR it now takes the Crusher 7 seconds to break contact from PVT Unga Bald (ok it's 6.9 seconds but humor me (also nice)) 

Now you might say; "one second isn't that long" In which case, you'd be insanely wrong. In this game, one second is massive. To show you. lets look at the DPS and total damage done in those few seconds. 

The M41 deals about 105 damage per second with a standard mag vs 30 armor. 

That means in the first case the Crusher will be dealt about 630 damage over the course of the engagement. Big ouch? Yes. Deadly? No. 

But in the second case. with that ONE extra second. The Crusher is now dealt 735 damage. The Crusher is now in crit. and if PVT Unga Bald still has eyes on him, the crusher is now dead. 

And that was with a SINGLE marine with a standard M41. What if you have his battle buddy shooting at you? suddenly that's an extra 210 damage in one second? OR what if he has 2 SGs with AP behind him. Thats an extra 505 damage. That's HALF a queen in that one second. What if there was more even? 3? 4, 7? Soon you are talking entire Healthpools + buffs gone in that ONE extra second. 

and this isn't even considering things like the HPR, M2C, M56D, Sniper aimmed shot, OT nades, CAS, Mortar, etc etc, 

All of these things roll for the marines favor, while Xenos (and even preds!) are literally left out to dry as 70% of the hive is meant for melee combat, and melee combat just got its shit kicked in by a 15% movement debuff. 



Look. I understand why and your reasons for this PR are just. But this game has been fine tuned to deal with fast based combat since MOBA, undoing it this quickly is simple going to rug pull Xenos, Preds and Dallas Weeter.

**Comment by Don2137** [comment]
> Look. I understand why and your reasons for this PR are just. But this game has been fine tuned to deal with fast based combat since MOBA, undoing it this quickly is simple going to rug pull Xenos, Preds and Dallas Weeter.

The game has been fine tuned before and the rug has been pulled before, more than once. It's perfectly doable as long as the dev team decides to commit to it.

**Comment by hotimep** [comment]
this pr sucks booty and the replies are bad takes city

**Comment by SarahJohnson489** [comment]
That's the problem Don. Who wants actual years of problems stemming from not committing to getting the tuning done and release the full rebalanced overhaul in its entirety and just barfing out a disaster.

**Comment by Don2137** [comment]
> That's the problem Don. Who wants actual years of problems stemming from not committing to getting the tuning done and release the full rebalanced overhaul in its entirety and just barfing out a disaster.

It should be obvious that any major change will come with growing pains. You can't redo half the game in a single PR. It's more important for the dev team to actually commit to a direction and stick to it, and I believe this change could be a good first step

**Comment by Don2137** [comment]
Personally, I'm a big fan of this PR. Every year CM gets faster, shorter, more streamlined, all the way from neth armor rework, RG prae meta, moba and techwebs. The CM of old was a "slowfast" game of sorts: the pace wasn't crazy but making a mistake could get you killed extremely quickly. It added a very exciting and visceral atmosphere to the game, like death was around every corner. In current CM, you can remove half of a xenos health before it can even slash you. It just doesn't feel the same.

I do believe the game could use a major effort to slow things down and make them more meaningful, and this PR is the first step. Every year we get closer to how TGMC is. Feels like a waste.

Speaking of roleplay specifically, since others are mentioning it: while a PR can't bring it back by itself, a slower game DOES allow more opportunities to talk to people. That's how it was and it could be like that again.

In general, yes, there are many changes that will need to be made to compensate for this PR, as others have said. But if devs still think that this game isn't just a TDM, like staff and some players believe, then this is 100% the right course of action.

**Comment by SarahJohnson489** [comment]
Which is why if such a thing will happen it's to be done as a proper project and get all the changes readied not haphazardly drag it out for 3 years then abandon it in a garbage state. You can't just run around shoving things over and shout growing pains like it absolves anything for those affected by it.
You already have 40-180+ minutes of opportunities to talk to people. They're right there to be spoken to and interacted with. That hasn't changed. What also hasn't changed is the list of things to do to help the round succeed. Taking longer means it's that much later to get done and take it easy. It means cutting things out of your routine and getting more 40 minute rounds as a result. It means losing RP unless the only RP you want to do is briefing, shuttle drop, memorial, or shipside rp.

**Comment by stanalbatross** [comment]
> I feel like this slowdown will hurt roleplay a lot. Right now you are unable to talk and walk, so getting left behind by your battlebuddies as you are talking a big issue (though typing indicators do help). This slowdown makes this issue even worse. If walking somewhere takes more time due to slowdown, than the opportunity cost of the movement you are giving up to type increased. tldr - time spent walking increased = time spent talking decreased

Facts my friend

**Comment by Bram-KEP** [comment]
People won't RP if you slow down the game they'll just break their W keys from slamming on it out of anger

**Comment by Bram-KEP** [comment]
People won't RP if you slow down the game they'll just break their W keys from slamming on it out of anger

**Comment by Binbers** [comment]
This is fr just everyone who used to play CM before 2020/MOBA who knows slowing the game = literally better VS everyone who started playing after CM became white noise™. I'll start playing the 2d spaceman war game again and be a marine main if this gets merged 🗿
